.github/workflows/e2e-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: make -f x.mk e2e-local NODES=2 JUNIT_DIRECTORY=./artifacts/
-      - name: Archive production artifacts # test results are only uploaded if any of the e2e tests fails
+      - name: Archive Test Artifacts # test results are only uploaded if any of the e2e tests fails
```

Comment:
Can this collect the test reports on success, too? That will help determine rates in terms of total test runs (e.g. overall suite success rate) over a period of time.

Reply:
I did think about it, but i went with not uploading successful test results in the first place. https://github.com/operator-framework/operator-lifecycle-manager/blob/f0495d925d62a80dc4a9abf6eb617b4150bc3f93/.github/workflows/e2e-tests.yml#L14

If we upload it, it will be collected by this code as well. I bet we could smoke out the flakes right there. same commit, passed and failed. But i am hoping to do that as a follow up after resolver thing?

Reply:
SGTM.